### PR TITLE
Release 11.0.0 with Kubernetes 1.16.3

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/7428

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -1,55 +1,48 @@
...
   authorities:
     - name: app-operator
       version: 1.0.0
-    - endpoint: http://aws-operator:8000
-      name: aws-operator
-      version: 7.1.0-dev
-    - endpoint: http://cert-operator:8000
-      name: cert-operator
+    - name: aws-operator
+      version: 7.1.0
```
[the remark this answers is not in the document]
Comment (tfussell) [reply aws.yaml]: Yes, but I haven't touched AWS yet because of all the complexity with the CAPI changes. I guess it will be 8.1 or 9.0.

**Comment by MarcelMue** [comment]
@cornelius-keller for proof reading

File: azure.yaml [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 2.0.0
```
Comment (tuommaki) [inline azure.yaml]: This might be wrong? Azure doesn't have Cluster API types yet.

Comment (MarcelMue) [reply azure.yaml]: Fixed.

**Comment by tfussell** [comment]
Closing in favor of separate provder-specific PRs (linked above).

**Pull request closed without merging**